detekt.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Provide detekt rules for Kotlin projects
 homepageUrl=https://github.com/detekt/sonar-kotlin
-archivedVersions=2.0.0,2.2.0
-publicVersions=2.1.0,2.3.0
+archivedVersions=2.0.0,2.2.0,2.3.0
+publicVersions=2.1.0,2.4.0
 
 defaults.mavenGroupId=io.github.detekt
 defaults.mavenArtifactId=sonar-detekt
 
+2.4.0.description=Upgrade to detekt 1.18.1
+2.4.0.sqVersions=[7.9.4,LATEST]
```

Comment:
Support for anything before 8.9 is officially over. This range needs to start with the current LTS: 8.9